src/camera_display.cpp

```diff
@@ -374,6 +379,50 @@ void CameraPub::updateFrameRate()
 {
 }
 
+void CameraPub::updateDisplayName()
+{
+    std::string name = name_property_->getStdString();
+    if (name.empty())
+    {
+        setStatus(StatusProperty::Warn, "Display name",
+                  "Should not be empty.");
+        return;
+    }
+
+    ros::NodeHandle nh_private_ = ros::NodeHandle("~"+name);
```

Comment:
Without the forced `~` prefix the user would able to select whether they want to use the tilde in the string property, no tilde (which moves up one namespace?), or put a leading slash for an absolute namespace.